cmd/kubeadm/app/phases/upgrade/compute_test.go

```diff
@@ -129,12 +129,12 @@ func TestGetAvailableUpgrades(t *testing.T) {
 		{
 			name: "no action needed, already up-to-date",
 			vg: &fakeVersionGetter{
-				clusterVersion: "v1.10.3",
-				kubeletVersion: "v1.10.3",
-				kubeadmVersion: "v1.10.3",
+				clusterVersion: "v1.13.0",
+				kubeletVersion: "v1.13.0",
+				kubeadmVersion: "v1.13.0",
 
-				stablePatchVersion: "v1.10.3",
-				stableVersion:      "v1.10.3",
+				stablePatchVersion: "v1.13.0",
+				stableVersion:      "v1.13.0",
```

Comment:
yeah, looks like we still have to bump a lot of versions, while constants constants should be used instead.
https://github.com/kubernetes/kubeadm/issues/1260